bitnami/orangehrm/README.md

```diff
@@ -52,114 +52,181 @@ The following table lists the configurable parameters of the OrangeHRM chart and
+| `readinessProbe`                     | Readiness probe configuration                                                                                         | `Check values.yaml file`                       |
+| `replicaCount`                       | Number of OrangeHRM Pods to run                                                                                       | `1`                                            |
+| `resources`                          | CPU/Memory resource requests/limits                                                                                   | Memory: `512Mi`, CPU: `300m`                   |
+| `sidecars`                           | Attach additional containers to the pod (evaluated as a template)                                                     | `nil`                                          |
+| `smtpHost`                           | SMTP host                                                                                                             | `nil`                                          |
+| `smtpPort`                           | SMTP port                                                                                                             | `nil`                                          |
+| `smtpProtocol`                       | SMTP Protocol (options: ssl, none)                                                                                    | `nil`                                          |
+| `smtpUser`                           | SMTP user                                                                                                             | `nil`                                          |
+| `smtpPassword`                       | SMTP password                                                                                                         | `nil`                                          |
+| `tolerations`                        | Tolerations for pod assignment                                                                                        | `[]` (The value is evaluated as a template)    |
+| `updateStrategy`                     | Deployment update strategy                                                                                            | `nil`                                          |
+
+### Traffic Exposure Parameters
+
+| Parameter                        | Description                             | Default           |
+|----------------------------------|-----------------------------------------|-------------------|
+| `service.type`                   | Kubernetes Service type                 | `LoadBalancer`    |
+| `service.port`                   | Service HTTP port                       | `80`              |
+| `service.httpsPort`              | Service HTTPS port                      | `443`             |
+| `service.externalTrafficPolicy`  | Enable client source IP preservation    | `Cluster`         |
```

Comment:
There seems to be two parameters missing in the table; `loadBalancerSourceRanges` and `loadBalancerIP`.